app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<h2>Sign up</h2>
+
+<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.input :first_name,
+                required: true,
+                autofocus: true,
+                input_html: { autocomplete: "first_name" }%>
+    <%= f.input :last_name,
+                required: true,
+                autofocus: true,
```

Comment:
The autofocus is an attribute that should only appear in one input. When the browser opens a page where there's a form with an autofocus input, it will automatically focus that element. Having more autofocus inputs is useless.

Reply:
Thanks Daniel! I generated a migration to change the default value of the boolean for :owner attribute to false and also removed the autofocus for email and pasword, leaving only the autofocus for first_name input field